dapps/marketplace/src/queries/Fragments.js

```diff
@@ -194,11 +194,19 @@ export default {
           status
           rewardEarned {
             amount
-            currency
+            currency {
+              ... on Currency {
```

Comment:
Hmm... we also need to change the back-end to return Currency since it got changed from Price to Currency. I can send a separate PR